persistence-cassandra/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/cassandra/CassandraConfigProvider.scala

```diff
@@ -29,15 +29,16 @@ final class CassandraConfigProvider @Inject() (system: ActorSystem) extends Prov
   override lazy val get: CassandraConfig = CassandraConfigProvider.CassandraConfigImpl(cassandraUrisFromConfig)
 
   private def cassandraUrisFromConfig: PSet[CassandraContactPoint] = {
-    val contactPoints = List("cassandra-journal", "cassandra-snapshot-store", "lagom.persistence.read-side.cassandra").flatMap { path =>
-      val c = config.getConfig(path)
-      if (c.getString("session-provider") == classOf[ServiceLocatorSessionProvider].getName) {
-        val name = c.getString("cluster-id")
-        val port = c.getInt("port")
-        val uri = new URI(s"tcp://127.0.0.1:$port/$name")
-        Some(CassandraContactPoint.of(name, uri))
-      } else None
-    }.toSet
+    val contactPoints: Set[CassandraContactPoint] =
+      List("cassandra-journal", "cassandra-snapshot-store", "lagom.persistence.read-side.cassandra").flatMap { path =>
+        val c = config.getConfig(path)
+        if (c.getString("session-provider") == classOf[ServiceLocatorSessionProvider].getName) {
+          val name = c.getString("cluster-id")
+          val port = c.getInt("port")
+          val uri = new URI(s"tcp://127.0.0.1:$port/$name")
+          Some(CassandraContactPoint.of(name, uri))
+        } else None
+      }.toSet
```

Comment:
As discussed over the phone, this seems dead code caused by 763b82987d9e69318b256e482bd1e03fc954a2a0